src/common.rs

```diff
@@ -0,0 +1,47 @@
+// Licensed under the Apache License, Version 2.0, <LICENSE-APACHE or
+// http://apache.org/licenses/LICENSE-2.0> or the MIT license <LICENSE-MIT or
+// http://opensource.org/licenses/MIT>, at your option. This file may not be
+// copied, modified, or distributed except according to those terms.
+
+use pdb::Error;
+use std::fmt::{self, Debug, Display, Formatter};
+use std::io;
+use std::result;
+use symbolic_minidump::cfi;
+
+#[derive(Debug)]
+pub enum DumpSymError {
```

Comment:
FWIW, if you're not planning to make this a library crate defining a full-fledged error enum doesn't really buy you much. You can just use `Box<dyn std::error::Error>` and things should work pretty well:
```rust
use std::error;

type Error = Box<dyn error::Error>;
type Result<T> = std::result::Result<T, Error>;
```

Reply:
Thanks for chiming in Ted, good to see you around :)

Reply:
Thanks for that and fixed.